### DISCUSSION TITLE
One click bookmrk

### DISCUSSION THREAD

**Discussion by m00ke5h** [dailydotdev/daily#930] [Feature Requests]
I find myself saving to bookmark many cards but currently, it takes two clicks to do that. It would be really great if the 'save to bookmarks' icon was on the card instead of inside the options button.

**Comment by idoshamun** [comment]
@m00ke5h thanks for the suggestion! We will consider

**Reply by nimrodkra** [reply]
@m00ke5h we added the bookmark button back to the card so you can save posts in one click - enjoy! 

![IMG_8444](https://github.com/dailydotdev/daily/assets/41470823/ecd81db5-46e9-4fe2-9a13-59da7df7d074)
